action.yml

```diff
@@ -8,6 +8,12 @@ inputs:
     description: "Path where the changeset files will be added"
     # required: true
     default: "changeset/fragments"
+  GH_APP_DOMAIN:
+    description: "GitHub App domain for API calls"
+    required: true
```

Comment:
We can hardcode that in the code for the `changelog workflow` (**Action Repository**). Thinking is not required to be passed from the **Consuming Repository**, but I want to check that. The important thing is to have two options for the `GH_APP_DOMAIN`: one for production and one for development. For the development one, we are suggesting using a `ngrok` tunnelling tool. We will detail that in the `DEVELOPER_GUIDE.md`

Reply:
At last, we decided to change the constant `GH_APP_DOMAIN` to `CHANGELOG_PR_BRIDGE_URL_DOMAIN`. After our discussion yesterday, we will keep both `CHANGELOG_PR_BRIDGE_URL_DOMAIN` and `CHANGELOG_PR_BRIDGE_API_KEY` as secrets sent by the **Consuming Repo** for the reusable action. In our roll-out, this would be OpenSearch-Dashboards repo

Reply:
Here the updated code

```yml
name: "Update Changelog"
description: "Update changelog file when a pull request is raised"
inputs:
  token:
    description: "GitHub token for authentication"
    required: true
  CHANGELOG_PR_BRIDGE_URL_DOMAIN:
    description: "URL domain for OpenSearch Chnagelog PR Bridge servie"
    required: true
  CHANGELOG_PR_BRIDGE_API_KEY:
    description: "API key for communicating for OpenSearch Changelog PR Bridge service"
    required: true
runs:
  using: "node16"
  main: "src/main.js"
```